tests/components/jewish_calendar/__init__.py

```diff
@@ -50,3 +52,13 @@ def make_jerusalem_test_params(dtime, results, havdalah_offset=0):
         JERUSALEM_LATLNG.lng,
         results,
     )
+
+
+@contextmanager
+def alter_time(retval):
+    """Manage multiple time mocks."""
+    patch1 = patch("homeassistant.util.dt.utcnow", return_value=retval)
+    patch2 = patch("homeassistant.util.dt.now", return_value=retval)
```

Comment:
Should we really patch both to the same time?

Reply:
Doesn't make a difference as they don't affect each other.

Reply:
Ok, but normally they would only return the same time if the local time was UTC, right. This doesn't matter for the tests?

Reply:
You're right. Normally they'd show different times. But it doesn't matter for the testing scenario, as dt.utcnow is only used for triggering the time change which causes async_update and dt.now is used as an argument to calculate the sensors values. I don't see why the sensor would ever use dt.utcnow. so they don't affect each other.

Reply:
Ok. :+1:

Reply:
Looks like I spoke to fast. It seems like now that I patched both, triggering of time change fails. I'll take a look at it tomorrow morning when I'm in front of a real machine. Could be due to the discrepancy between utcnow and now.